aries-site/src/examples/components/datatable/DataTableVerticalListExample.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button, 
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import ServerGroups from '../../../data/mockData/serverGroups.json';
+
+/* This function displays an arr's values in a vertical list within a table 
+   cell, hiding all values from view that exceed the given maximum list 
+   length until the "View More" button is clicked, at which point the 
+   button label changes to "View Less". */
+function ViewMore(arr, maxParam) {
+    const [viewMoreBool, setViewMoreBool] = useState(false);
+    return (
+        <Box align="end">
+            <Text truncate>{viewMoreBool ? arr.map(tmp => 
+                <Box>
+                    <Text>{tmp}</Text>
+                </Box>) : arr.map(tmp => 
+                <Box>
+                    <Text>{tmp}</Text>
+                </Box>).slice(0, maxParam)
+                }
+            </Text>
+            {arr.length > maxParam && (
+                <Button 
```

Comment:
It looks like ESLint / Prettier is not being applied here. Can you try the following steps in VSCode (we can connect on Slack if this doesn't work):
- In "Extensions" add "ESLint" and "Prettier"
- Quit out of VSCode
- Open the design system repo in VSCode
- Go to your file and hit "Save" -- some of the code should reformat.